### PR TITLE
Adds Netty + Google gRPC example

### PR DESCRIPTION
**Pull request by adriancole**
Screen Shot 2020-12-19 at 2 43 33 PM

cc also @saturnism

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ Here are the example projects you can try:
   * Trace Instrumentation: [Jersey Server](https://github.com/openzipkin/brave/tree/master/instrumentation/jersey-server), [DataStax Java Driver](https://github.com/openzipkin/brave-cassandra/tree/master/cassandra-driver), [Apache Cassandra](https://github.com/openzipkin/brave-cassandra/tree/master/cassandra), [SLF4J](https://github.com/openzipkin/brave/tree/master/context/slf4j)
   * Trace Configuration: [Brave API](https://github.com/openzipkin/brave/tree/master/brave#setup) [XML](jersey2-cassandra3/src/main/webapp/WEB-INF/tracing.xml)
 
+* [netty4-grpc](netty4-grpc) `BRAVE_EXAMPLE=netty4-grpc docker-compose up`
+  * Runtime: Netty 4.1, Google gRPC 1.31, SLF4J 1.7, JRE 15
+  * Trace Instrumentation: [Netty Codec HTTP](https://github.com/openzipkin/brave/tree/master/instrumentation/brave-instrumentation-netty-codec-http), [Google gRPC](https://github.com/openzipkin/brave-cassandra/tree/master/cassandra-grpc), [SLF4J](https://github.com/openzipkin/brave/tree/master/context/slf4j)
```
Comment (adriancole) [inline README.md:56]:
```suggestion
  * Trace Instrumentation: [Netty Codec HTTP](https://github.com/openzipkin/brave/tree/master/instrumentation/netty-codec-http), [Google gRPC](https://github.com/openzipkin/brave/tree/master/instrumentation/grpc), [SLF4J](https://github.com/openzipkin/brave/tree/master/context/slf4j)
```

**Comment by adriancole** [comment]
hmm I don't think the GH actions syntax is working as all projects built even though I only changed this folder (in the last commit) I wonder if it is considering all commits on the branch or if the syntax is broke.

**Comment by adriancole** [comment]
wild guess is that if the branch effects all projects, it will build all projects on each commit, even if later commits don't effect all projects. I tested with https://github.com/openzipkin/brave-example/pull/84 which shows it works if you only touch one project

**Review by anuraaga — APPROVED** [review]

**Comment by adriancole** [comment]
found that overall everything is smaller in terms of lines and image size to not use spring. reverted for now until we find a lighter config option for things which otherwise have no opinion.

**Pull request merged**